### DISCUSSION TITLE
Why has the physical size of the Print UI changed for this user?

### DISCUSSION THREAD

**Discussion by ajtruckle** [MicrosoftEdge/WebView2Feedback#4042] [Q&A]
One of my users is having issues with the Print UI window. When they were using my previous version of my software the window looked OK:

<img width="326" alt="Print1" src="https://user-images.githubusercontent.com/7985331/235468507-be72fe83-f470-4241-bda6-d40af22cc934.png">

Now, after upgrading my software the Print UI looks different:

<img width="449" alt="Print2" src="https://user-images.githubusercontent.com/7985331/235468766-23ee637a-1411-4751-803b-a7e2290aa6d8.png">

I did update the WebView2 NuGet package between releases of my software. Now he is using:

```
WebView2 Runtime Version: 112.0.1722.64
WebView2 SDK Build: 1722.45
```

In my software I have an option to display the document in a popup window which is maximized to the users display. I got him to do this, and then right-click, and select Print from there. The Print UI looks better:

![Print3](https://user-images.githubusercontent.com/7985331/235469287-55713719-6da5-4a37-b596-301fa8a4ee5e.jpg)

My question is, why has the physical width of the Print UI panel changed between releases of the WebView2 SDK in my application on his computer? I don't get this issue on my own PC. What is causing this problem?

**Comment by monica-ch** [comment]
@ajtruckle Sorry your user has the issue with Print UI size. I am not seeing the issue in our sample apps with different runtime versions. Has your user tried to restart the app and machine? Can you also ask them to delete UDF folder and launch the app again to see if issue still persists.

**Comment by ajtruckle** [comment]
@monica-ch I have asked him these questions. Where exactly is the UDF folder though?

**Comment by monica-ch** [comment]
It is user data folder to store browser data host app https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/user-data-folder?tabs=win32

**Comment by ajtruckle** [comment]
@monica-ch Ouch, my application is using demo code!

<img width="601" alt="Screenshot 2023-05-03 at 04 53 44" src="https://user-images.githubusercontent.com/7985331/235831375-b769fc96-ef0b-4879-bab1-069e3f0079c8.png">

I will have to correct that!

**Comment by ajtruckle** [comment]
The sample application uses the local application data folder as the root. Whereas I use the user application data folder as the root of my application settings. Eg:

```
CString CMeetingScheduleAssistantApp::GetWorkingPath()
{
	CString	strPath;

	SHGetSpecialFolderPath(nullptr, strPath.GetBuffer(_MAX_PATH), CSIDL_APPDATA, FALSE);
	strPath.ReleaseBuffer();

	if (strPath.Right(1) != _T("\\"))
		strPath += _T("\\");
	strPath += _T("Meeting Schedule Assistant\\");

	return strPath;
}
```
This would be path that I would pass into WebView2 now. But I have been reading up on the difference between Local and Roaming. I assume it is better that I use Local for the WebView2 data … because of it's size? 🤔

**Comment by ajtruckle** [comment]
@monica-ch I connected to the users computer today with AnyDesk and deleted the UDF and it made no 
difference. But I did a comparison of his / my display settings. These are my settings:

![Scale](https://user-images.githubusercontent.com/7985331/236274709-6a52f190-aa98-4c7a-883e-9fe5301a4906.png)

* On my system the text scale setting is **100% (recommended)**.
* On his system (laptop) the text scale setting is **150% (recommended)**.
 
If I set the text scale to **100%** then the Print UI looks "normal" (from my point of view) with the correct width for the side panel. But the issue is that my application (MFC Dialog) is then quite small so it is not the best scale factor to use.

Reverting back to the operating system's recommended 150% text scale causes the large Print UI side panel again.

* Screen resolution: 1920 x 1080
* Screen size: 13.3 inch (diagonals)

Does any of this help?

**Comment by ajtruckle** [comment]
@monica-ch Any comments on my observations?

**Comment by monica-ch** [comment]
@ajtruckle What was changed in the newsiest version of your software other than wv2 sdk version ?
